ui/admin-portal/src/app/components/candidates/show/show-candidates.component.ts

```diff
@@ -212,7 +213,8 @@ export class ShowCandidatesComponent implements OnInit, OnChanges, OnDestroy {
               private router: Router,
               private candidateSourceResultsCacheService: CandidateSourceResultsCacheService,
               private candidateFieldService: CandidateFieldService,
-              private authService: AuthService,
+              private authService: AuthorizationService,
```

Comment:
Maybe good to rename authService instance variables throughout to authorizationService